build.sbt

```diff
@@ -46,6 +46,7 @@ lazy val commonSettings = Seq(
   organization in ThisBuild := "io.github.jmcardon",
   scalaVersion in ThisBuild := "2.12.4",
   fork in test := true,
+  fork in run := true,
```

Comment:
Yes, because I was testing stuff by using the `run` command and sbt doesn't release classloaders for native links, making multiple `run`s useless without this.

Hell, I think eventually when a PR comes for the `libsodium` module, i think that we should note to users that forking the jvm is the only way to run things in sbt that won't break and require a restart every two seconds :|. It's shitty, but try commenting it out, running some JNI code and seeing what happens. The results suck.